### PR TITLE
interfaces/avahi*: update avahi-daemon labelling to also allow "avahi-daemon"

### PR DESCRIPTION
**Pull request by jetpackdanger**
This appears to be the label used by avahi-daemon on Ubuntu 20.04 LTS.

Confirmed by patching locally that this lets a confined snap I'm working on successfully register with Avahi.

### CODE REVIEW

**Comment by jhenstridge** [comment]
Presumably this would be running with the following profile:

https://gitlab.com/apparmor/apparmor/-/blob/master/profiles/apparmor.d/usr.sbin.avahi-daemon

... which does indeed use the profile name `avahi-daemon`.  I wonder if there actually are any profiles called `/usr/sbin/avahi-daemon`?

**Comment by bboozzoo** [comment]
> Presumably this would be running with the following profile:
> 
> https://gitlab.com/apparmor/apparmor/-/blob/master/profiles/apparmor.d/usr.sbin.avahi-daemon
> 
> ... which does indeed use the profile name `avahi-daemon`. I wonder if there actually are any profiles called `/usr/sbin/avahi-daemon`?

Looks like this commit introduced the change https://gitlab.com/apparmor/apparmor/-/commit/b77116e6afeec7b7bd366c2d74ddf225fb1dfeb3 On the bright side, it does not look like we need to update anything else.

File: interfaces/builtin/avahi_control.go [hunk 1]
```diff
@@ -123,7 +123,7 @@ func (iface *avahiControlInterface) AppArmorConnectedPlug(spec *apparmor.Specifi
 	// label like it would when installed as a snap.
 	if implicitSystemConnectedSlot(slot) {
 		// avahi from the OS is typically unconfined but known to sometimes be confined
-		new = "\"{unconfined,/usr/sbin/avahi-daemon}\""
+		new = "\"{unconfined,/usr/sbin/avahi-daemon,avahi-daemon}\""
```
Comment (bboozzoo) [inline interfaces/builtin/avahi_control.go:126]: Could use a comment:
```
// with stock apparmor 2.13.2+ profiles the label is avahi-daemon
```

Comment (jetpackdanger) [reply interfaces/builtin/avahi_control.go:127]: Re-pushed with comment added. Thanks for tracking this down!

**Pull request merged**